fabric/fabric-utils/src/main/java/io/fabric8/utils/FabricValidations.java

```diff
@@ -90,6 +109,10 @@ public static boolean isValidProfileName(String name) {
     public static boolean isValidVersionName(String name) {
         return name != null && !name.isEmpty() && ALLOWED_VERSION_NAMES_PATTERN.matcher(name).matches();
      }
+    
+    public static boolean isValidPidProperty(String pidProperty){
+        return pidProperty != null && !pidProperty.isEmpty() &&  (!pidProperty.contains(PID_KEY_SEPARATOR)) || (pidProperty.contains(PID_KEY_SEPARATOR) && PID_PROPERTY_PATTERN.matcher(pidProperty).matches());
```

Comment:
I think this should be something like:
```java
return pidProperty != null && !pidProperty.isEmpty() &&  ( !pidProperty.contains(PID_KEY_SEPARATOR) ||  PID_PROPERTY_PATTERN.matcher(pidProperty).matches() );
```
otherwise if `pidProperty != null && !pidProperty.isEmpty() &&  (!pidProperty.contains(PID_KEY_SEPARATOR))` evaluates to `false`, then `(pidProperty.contains(PID_KEY_SEPARATOR) && PID_PROPERTY_PATTERN.matcher(pidProperty).matches())` would still be checked, which I think wasn't the intended behaviour ? 
Have I understood correctly the intention of the check?